src/helpers/formatting.ts

```diff
@@ -0,0 +1,15 @@
+import { padStart, reverse } from 'lodash-es';
+
+export function formatDuration(numSeconds: number | null | undefined) {

```

Comment:
`formatDuration(numSeconds?: number)`
En dan een check toevoegen in de functie zelf `if(numSeconds)`

Also, ik zou voor `seconds` gaan ipv. `numSeconds`, het zou duidelijk moeten zijn uit de typing-informatie dat het om een `number gaat`.

Reply:
Eigenlijk zou ik zelfs die parameter niet undeclared laten zijn? Zonder kan die functie toch niet werken?